lib/govspeak.rb

```diff
@@ -32,7 +32,7 @@ def self.to_html(source, options = {})
     end
 
     def initialize(source, options = {})
-      options.deep_symbolize_keys!
+      options = options.dup.deep_symbolize_keys
```

Comment:
Nah, the assignment doesn't mutate the original object. As that assignment has no effect outside of this lexical scope. Not sure I explained this well though.